src/devices/bus/isa/sblaster_lle.h

```diff
@@ -0,0 +1,91 @@
+// license:BSD-3-Clause
+// copyright-holders:Katherine Rohl
+#ifndef MAME_BUS_ISA_SBLASTER_LLE_H
+#define MAME_BUS_ISA_SBLASTER_LLE_H
+
+#pragma once
+
+#include "isa.h"
+#include "bus/pc_joy/pc_joy.h"
+#include "cpu/mcs51/mcs51.h"
+#include "sound/dac.h"
+#include "sound/ymopl.h"
+#include "sound/spkrdev.h"
+#include "speaker.h"
```

Comment:
Do you need to `#include "speaker.h"` here?  I think it’s only needed in the implementation file.